lib/cartopy/mpl/geoaxes.py

```diff
@@ -425,14 +425,17 @@ def coastlines(self, resolution='110m', color='black', **kwargs):
         ----------
         resolution
             A named resolution to use from the Natural Earth
-            dataset. Currently can be one of "110m", "50m", and "10m".
+            dataset. Currently can be one of "110m", "50m", and "10m",
+            or a Scaler object.
 
         """
         kwargs['edgecolor'] = color
         kwargs['facecolor'] = 'none'
-        feature = cartopy.feature.NaturalEarthFeature('physical', 'coastline',
-                                                      resolution, **kwargs)
-        return self.add_feature(feature)
+        feature = cartopy.feature.COASTLINE
+        if resolution != 'auto':
```

Comment:
This if block doesn't do anything currently (I think you have a PR in the wings waiting for this to be implemented though...)

Reply:
Yes this is for an upcoming PR. Should I remove it from this one and commit it to the next one?